src/ContainerTerminal.jsx

```diff
@@ -246,18 +248,9 @@ class ContainerTerminal extends React.Component {
     render() {
         let element = <div className="container-terminal" ref="terminal" />;
         if (this.state.errorMessage)
-            element = (<div ref="terminal" className="empty-state">
-                <span className="empty-message">
-                    { this.state.errorMessage }
-                </span>
-            </div>
-            );
-
-        return (
-            <>
-                { element }
-            </>
-        );
+            element = <EmptyStatePanel icon={ExclamationCircleIcon} title={this.state.errorMessage} />;
```

Comment:
Also I would say that this does not need an ExclamationCircleIcon since it's not an error here, I would just say something like VM page does.
![Screen Shot 2020-11-27 at 16 54 57](https://user-images.githubusercontent.com/14921356/100466508-4e023e00-30d1-11eb-8f6f-0b515cf4dc30.png)

Reply:
![noptrunning20](https://user-images.githubusercontent.com/12330670/100474394-cb817a80-30e0-11eb-96c6-9380b4a73fbc.png)